ha/alert/alert_monitor.py

```diff
@@ -62,32 +64,28 @@ def process_alert(self):
         """
         Process alert
         """
-
         self._validate_pacemaker()
         self._load_recipient()
 
         # validate event type and get class
         try:
-             event_type = self._validate_event()
```

Comment:
Remove the function definition for _validate_event since its unused now

Reply:
done